crates/pixi_build_frontend/src/protocol_builder.rs

```diff
@@ -5,10 +5,32 @@ use rattler_conda_types::ChannelConfig;
 use crate::{
     conda_build_protocol, pixi_protocol,
     protocol::{DiscoveryError, FinishError},
+    rattler_build_protocol,
     tool::ToolCache,
     BackendOverride, BuildFrontendError, Protocol,
 };
 
+#[derive(Debug)]
+pub struct DiscoveryConfig {
```

Comment:
Needs docs, and not sure I like the name `DiscoveryConfig` because it seems to enable Protocols. I know this influences discovery but feels like implementation detail. Maybe something like `EnabledProtocolImplementations` or something shorter?